README.md

```diff
@@ -7,23 +7,27 @@
   - https://github.com/HiveFunder/support-module-svc
   - https://github.com/HiveFunder/updates-service
 
+## Setup Instructions
+To seed data for this project, first run `node db/largeDataGenAndSeed.js`, then run the bash shell script `./db/mongoSeed.sh`, both of which from the project root.
+If your laptop has an SSD, it should take no more than 30 minutes to generate and seed everything!
+
 ## CRUD Documentation
 
 ### Data Shape
-  - Campaign: Campaign name, description, author, currency, pledge amount, pledge goal, backers, end date, type, location
+  - Campaign: Campaign name, description, author, username, country, pledge amount, pledge goal, backers, end date, type
   - Pledge: username, pledge amount, date
 
 ### Page Endpoint
-/:author/:campaignId/
+/:user/:campaignId/
 
 ### GET - /api/:campaign/stats
   - Get stats from a single campaign, for an item detail view.
   - Query format: Campaign ID (number)
   - Response data shape: Refer to Campaign data shape above
   - Error code: 400 for invalid author or campaign
 
-### GET - /api/:author/campaigns
-  - Get all campaigns for given author
+### GET - /api/:user/campaigns
```

Comment:
I like using /api/. Should we make our routes use this naming convention?

Reply:
Yes, I believe using /api/ path to refer to an API is common. I know NASA [does it](https://eonet.sci.gsfc.nasa.gov/docs/v2.1#eventsAPI)!